site/docs/latest/admin/http.md

```diff
@@ -0,0 +1,317 @@
+---
+title: BookKeeper http endpoints
+---
+
+This document is to introducing BookKeeper http endpoints that could be used for BookKeeper administration.
```

Comment:
This document introduces BookKeeper HTTP endpoints, which can be used ...

Reply:
Thanks, will change it.